src/test/java/uk/gov/justice/digital/delius/integration/secure/OffendersResource_getOffenderNsisByCrn.java

```diff
@@ -0,0 +1,113 @@
+package uk.gov.justice.digital.delius.integration.secure;
+
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.restassured.RestAssured;
+import io.restassured.config.ObjectMapperConfig;
+import io.restassured.config.RestAssuredConfig;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import uk.gov.justice.digital.delius.data.api.NsiWrapper;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@ActiveProfiles("dev-seed")
+@RunWith(SpringJUnit4ClassRunner.class)
+public class OffendersResource_getOffenderNsisByCrn {
+
+    private static final String OFFENDERS_PATH = "/offenders/crn/%s/convictions/%s/nsis/%s";
```

Comment:
Is a path param an appropriate way of passing the NSI type? I'd expect this to be a queryParam as it's a filter?

Reply:
I think I agree, query param makes more sense

Reply:
As it happens there's another problem I've stumbled across here whilst I've been updating [PIC-389](https://dsdmoj.atlassian.net/browse/PIC-389) to bring it in line with the concept of an NSI rather than a breach as the resource. That would give us a clash between this and the url for a single NSI which would end up like this: `secure/offenders/crn/{crn}/convictions/{convictionId}/nsis/{nsiId}`

Reply:
Yes - agreed Query Param :-)